client-js/TUTORIAL.md

```diff
@@ -0,0 +1,208 @@
+# CEP-78 JavaScript Client Tutorial
+
+This tutorial outlines the usage of the JavaScript client available for the CEP-78 Enhanced NFT Standard.
+
+Further information on the CEP-78 Enhanced NFT Standard can be found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+The client is available in the npm as [casper-cep78-js-client](https://www.npmjs.com/package/casper-cep78-js-client).
+
+## Client Installation
+
+The client can be installed in a project you have built using TypeScript / Javascript.
+
+To install run:
+
+```js
+npm install casper-cep78-js-client
+```
+
+## Installing a CEP-78 Contract using the JavaScript Client
+
+The `install` method crafts a [Deploy](https://docs.casperlabs.io/design/casper-design/#execution-semantics-deploys) using `InstallArgs`.
+As every deploy created by the SDK you can send it using `.send(rpcUrl)` method providing rpc url that you want to use. It will return deployHash. 
+
+```js
+
+  const contractClient  = new CEP78Client(process.env.NODE_URL!, process.env.NETWORK_NAME!);
+
+  const installDeploy = await contractClient.install(
+    {
+      collectionName: "my-collection",
+      collectionSymbol: "MY-NFTS",
+      totalTokenSupply: "1000",
+      ownershipMode: NFTOwnershipMode.Transferable,
+      nftKind: NFTKind.Physical,
+      jsonSchema: {
+        properties: {
+          color: { name: "color", description: "", required: true },
+          size: { name: "size", description: "", required: true },
+          material: { name: "material", description: "", required: true },
+          condition: { name: "condition", description: "", required: false },
+        },
+      },
+      nftMetadataKind: NFTMetadataKind.CustomValidated,
+      identifierMode: NFTIdentifierMode.Ordinal,
+      metadataMutability: MetadataMutability.Immutable,
+      mintingMode: MintingMode.Installer,
```

Comment:
This doesn't include the more recent modalities like burn or reverse lookup, I think these are the required args. Lets call out that some args are optional and have default values

Reply:
https://github.com/casper-ecosystem/cep-78-enhanced-nft/blob/dev/client-js/examples/install.ts#L41

Reply:
Example is updated to add `ownerReverseLookupMode`, and the list below includes optional parameters and what they default to.